doc/developers/contributing.rst

```diff
@@ -791,7 +804,11 @@ We are glad to accept any sort of documentation:
       SelectKBest : Select features based on the k highest scores.
       SelectFpr : Select features based on a false positive rate test.
 
-  * Add one or two snippets of code in "Example" section to show how it can be used.
+  * The "Notes" section is optional. It is meant to provide information on
+    specific behavior of the class/classmethod/method.
+
+  * Add one or two **snippets** of code in "Example" section to show how it can
```

Comment:
Maybe we should add that the code should be runable as is - i.e. should include all required imports?